features/buyers/account_plans/edit.feature

```diff
@@ -0,0 +1,44 @@
+@javascript
+Feature: Account plan edit page
+
+  Background:
+    Given a provider is logged in
+    And the provider has "account_plans" allowed
+    And the following account plan:
+      | Issuer               | Name |
+      | foo.3scale.localhost | Free |
+
+  Scenario: Navigation
+    Given they go to the account plans admin page
+    When they follow "Free" within the table
+    Then the current page is account plan "Free" admin edit page
+
+  Scenario: Edit an account plan
+    Given they go to account plan "Free" admin edit page
+    And they should see "Account plan Free"
+    When the form is submitted with:
+      | Name                       | Still free |
+      | Accounts require approval? | Yes        |
+    Then the current page is the account plans admin page
+    # And they should see "Plan was updated"
+    And the table has the following row:
```

Comment:
Maybe verify that the "account approval" field has been updated successfully?

Reply:
The idea is to test all inputs, therefore the checkbox has to be there. I did not want to test any "background" changes in the scenario, only UI. Ideally we should test that a new account using this plan will require approval..? I will add a step for that.

Reply:
I am not referring to "background changes". And not checking that the "approval" checkbox really works as intended (although we can test it - in another scenario).
What I mean is to test that if we check the checkbox and submit the form - this change is actually applied.

Like with the name:
1. Verify that the name is "Free"
2. Submit the form with "Still free"
3. After the page is reloaded, you can see that the name of the plan is "Still free"

So, I was looking for something like:
1. Verify that the approval checkbox in uncheckd
2. Check it and submit the form
3. After the page is reloaded, verify that this plan now has the approval checked

I guess the inconvenience here is that we're redirected to index after submitting the form, so the checkbox is not visible. We can either go to the edit page again to confirm that, or.... ignore this :grimacing: